webpack.config.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        exclude: /node_modules/,
```

Comment:
Ummm. Unless I'm mistaken, that's not built into Webpack. Webpack will run loaders on all matching files regardless of whether it's in node_ modules. Babel may default to not transpile node_modules though. That could make sense I guess...